packages/native-federation-typescript/src/rpc/wrap-rpc.ts

```diff
@@ -99,11 +87,18 @@ export function wrapRpc<T extends (...args: any[]) => any>(
     };
 
     // add event listeners
-    childProcess.on('message', handleMessage);
-    childProcess.on('close', handleClose);
+    if (once) {
```

Comment:
I'm not sure if message needs to follow the once strategy here @2heal1

Reply:
Not sure, i will check this after vacation